src/helpers/getRecord.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * Find Aha! single record from search string
+ *
+ * @param searchStr
+ * @returns
+ */
+export const getRecord = async (searchStr: string): Promise<Aha.RecordUnion | null> => {
+  const reference = extractStrToReference(searchStr);
```

Comment:
This needs a `try/catch` – Azure DevOps doesn't provide as many details in the webhook response when the build is manually triggered.

```
ETypeError: Cannot read property 'match' of undefined
    at extractStrToReference (src/helpers/getRecord.ts:58:23)
    at getRecord (src/helpers/getRecord.ts:8:21)
    at handleBuildCompleted (src/endpoints/azurePipelines.ts:35:22)
    at src/endpoints/azurePipelines.ts:13:13
    at Aha.callEndpoint (ContributionRegistry.js:161:16)
    at globalThis.globalCallEndpoint (server.js:66:7)
```